week2homework.js

```diff
@@ -0,0 +1,121 @@
+//1
```

Comment:
I see that you did all your homework in one file, but the assignment was to make one file for each part of the homework:

> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference.

Reply:
Thank you for your review, I'v made changes as advised, could you please re-check?